core/src/main/scala/kafka/tools/ConsoleProducer.scala

```diff
@@ -237,6 +241,10 @@ object ConsoleProducer {
             .withRequiredArg
             .describedAs("producer_prop")
             .ofType(classOf[String])
+    val producerConfigOpt = parser.accepts("producer.config", "Producer config properties file.")
```

Comment:
Can we use `producer-config` instead of  `producer.config` to be inline with format of other options?